app/jobs/get_video_metadata_job.rb

```diff
@@ -25,6 +24,8 @@ def perform(link)
     if @lt.images.count > 0
       img = @lt.images.first
       @thumbnail.source = img.src
+      # Is this really needed?
+      # Why are we dowloading the file to public?
```

Comment:
@Vexxenn I have no idea. Is there a better way to do this? We can change this since it's happening in a container and ultimately being deleted on the next deploy anyway.

Reply:
@elubow The image should be in S3 since the src url contained bjjdb-test.s3. In regards to
```
dir_path = File.join Rails.root, 'public', 'downloads', 'users', link.user_id.to_s, 'images'
FileUtils.mkdir_p(dir_path) unless File.exist?(dir_path)
download_image(img.src, File.join(dir_path, (URI(img.src).path).split('/').last))
```
i don't know if its necessary since its only creating a copy of the thumbnail.